template/helper/forkOnGithub.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+final class forkOnGithub
+{
+    private $githubRepo;
+
+    public function __construct(\Bookdown\Bookdown\Config\IndexConfig $config)
+    {
+        $default = 'https://github.com/prooph/proophessor';
```

Comment:
How about assigning this as a default value to `$githubRepo`, and then, instead of passing `$default` to `extractGithubRepo()`, adjusting the latter to modify `$githubRepo` when it was able to extract it from the configuration file path?